_test/general.test.php

```diff
@@ -58,4 +78,408 @@ public function test_plugin_conf() {
         }
 
     }
+
+    //======================================================================
+    // XML TESTS
+    //======================================================================
+
+    /**
+     * Test to ensure that XML is valid when DokuWiki has no pages.
+     */
+    public function test_xml_response_is_valid_if_dokuwiki_has_no_pages() {
+        $conf = array();
+        $DokuwikiXMLExport = new DokuwikiXMLExport($conf);
+        $xml = new SimpleXMLElement($DokuwikiXMLExport->generateXMLExport(0, 20));
+
+        $start = implode('', ($xml->xpath('/findologic/items/@start')));
```

Comment:
Nice solution for getting the actual value from `xpath()`!

Reply:
Thanks :)